### ISSUE TITLE
[PM] 10-10EZ - Improvement: Evaluate the impact of prompting for authentication

### ISSUE THREAD

**Issue by hdjustice** [department-of-veterans-affairs/va.gov-team#51633]
## Background
evaluate impact of pushing users down login path for 1010EZ. basically, can we gather sufficient information to decide whether or not it make sense to stick with the current flow that pushes people to log in before starting the form, and if so, does it make sense to reevaluate?

**Comment by hdjustice** [comment]
Closing - duplicate of #41570

**Comment by hdjustice** [comment]
meeting 1/30/2023 - test hypothesis that users will have a better experience when logged in to 10-10EZ.  Historically, ~3 of 10 fail log in process.  Looking to understand landscape there, and whether it is clear that signing in doesn't have market value for completing the form.  Let's get the data & do the analysis.  
Event tracking may have this data.
